### PR TITLE
refactor(txpool): Use Geth TxPool

### PR DESCRIPTION
**Pull request by itsdevbear**

### CODE REVIEW

File: cosmos/x/evm/keeper/abci.go [hunk 1]
```diff
@@ -24,17 +24,36 @@ import (
...
+	if err := k.polaris.Finalize(ctx); err != nil {
+		return err
+	}
+
+	// Prepare the txpool for the next pending block.
+	// TODO: move this.
+	chainConfig := k.host.GetConfigurationPlugin().ChainConfig()
+	k.GetHost().GetTxPoolPlugin().Prepare(
+		misc.CalcBaseFee(chainConfig, k.polaris.Blockchain().CurrentBlock()),
+		coretypes.LatestSignerForChainID(chainConfig.ChainID),
```
Comment (calbera) [inline cosmos/x/evm/keeper/abci.go:55]: use MakeSigner, you have the block number and time

Comment (calbera) [reply cosmos/x/evm/keeper/abci.go:55]: also why not just call this in chain writer

Comment (itsdevbear) [reply cosmos/x/evm/keeper/abci.go:55]: dont wanna give blockchain access to the tx pool plugin in preparation for breaking it out properly

File: eth/polar/backend.go [hunk 2]
```diff
@@ -441,51 +430,56 @@ func (b *backend) SubscribeChainSideEvent(ch chan<- core.ChainSideEvent) event.S
 // Transaction Pool API
 // ==============================================================================
 
-func (b *backend) SendTx(ctx context.Context, signedTx *types.Transaction) error {
-	return b.polar.blockchain.SendTx(ctx, signedTx)
+func (b *backend) SendTx(_ context.Context, signedTx *types.Transaction) error {
+	return b.polar.txPool.AddRemotes([]*types.Transaction{signedTx})[0]
```
Comment (calbera) [inline eth/polar/backend.go:434]: `AddLocal`

**Review by ocnc — DISMISSED** [review]
LGTM, it survived the load test

**Comment by itsdevbear** [comment]
closing in favour of #1140

**Pull request closed without merging**